### PR TITLE
Enable GC spectest for NuttX

### PR DESCRIPTION
**Pull request by no1wudi**

### CODE REVIEW

**Review by wenyongh — APPROVED** [review]
LGTM

File: .github/workflows/spec_test_on_nuttx.yml [hunk 1]
```diff
@@ -75,20 +81,25 @@ jobs:
...
+          #   option: "CONFIG_INTERPRETERS_WAMR_AOT=y\\n"
+          # },
+          # {
+          #   mode: "-t classic-interp",
+          #   option: "CONFIG_INTERPRETERS_WAMR_CLASSIC=y\\n"
+          # },
+          # {
+          #   mode: "-t fast-interp",
+          #   option: "CONFIG_INTERPRETERS_WAMR_FAST=y\\n"
+          # },
```
Comment (yamt) [inline .github/workflows/spec_test_on_nuttx.yml:95]: @no1wudi 
why did you comment these out?

Comment (no1wudi) [reply .github/workflows/spec_test_on_nuttx.yml:95]: There may have some potential bug, spectest for interpreter is extremly slow, about 4H.

Comment (yamt) [reply .github/workflows/spec_test_on_nuttx.yml:95]: do you mean GC-related bugs?

Comment (no1wudi) [reply .github/workflows/spec_test_on_nuttx.yml:95]: No, I think this bug is from test script, No-gc case also very slow, with GC enabled the test maybe over 6H that exceed the limit of GA runner

**Pull request merged**